lib/DAV/WorkspacePlugin.php

```diff
@@ -96,7 +96,7 @@ public function propFind(PropFind $propFind, INode $node) {
 				try {
 					$cachedContent = $file->getContent();
 					$cache->set($cacheKey, $cachedContent, 3600);
-				} catch (GenericFileException|NotPermittedException|LockedException $e) {
+				} catch (\Exception $e) {
```

Comment:
nit: shall we log this, at least with debug level?

Reply:
Changed to still ignore file, permission and locking exception but log anything else as error